src/ServerNotifications/AppStoreServerNotification.php

```diff
@@ -46,7 +46,7 @@ public function getSubscription(): SubscriptionContract
      */
     public function isTest(): bool
     {
-        return false;
+        return ServerNotification::ENV_SANDBOX === $this->notification->environment;
```

Comment:
@oleksandr-roskovynskyi 
Did you check the [access modifier](https://github.com/oleksandr-roskovynskyi/appstore-iap/blob/main/src/ServerNotifications/ServerNotification.php#L39) of the `$environment` property?